lib/tasks/hyacinth.rake

```diff
@@ -1,4 +1,10 @@
 # frozen_string_literal: true
 
 namespace :hyacinth do
+  task reindex: :environment do
+    DigitalObjectRecord.find_in_batches(batch_size: 100) do |records|
```

Comment:
@barmintor We could probably bump this up to `batch_size: 1000` if you and @cgalarza don't mind.

Reply:
```suggestion
    DigitalObjectRecord.find_in_batches(batch_size: (ENV['BATCH_SIZE'] || 1000).to_i) do |records|
```

Reply:
How do you feel about the suggested change?

Reply:
Perfect! 👍